src/overlays/actors/ovl_Obj_Spinyroll/z_obj_spinyroll.h

```diff
@@ -34,26 +34,26 @@ typedef struct {
 } ObjSpinyrollStruct2; // size = 0xF8?
```

Comment:
?

Reply:
?? :) (see the alternative testing piece, still need to figure what these structs actually are)